controllers/cache_handler.go

```diff
@@ -123,6 +123,12 @@ func (h DisruptionTargetWatcherHandler) OnChangeHandleNotifierSink(oldPod, newPo
 			eventType = corev1.EventTypeNormal
 		}
 
+		if eventReason == chaosv1beta1.EventReadinessProbeChange {
+			if h.disruption.Status.InjectionStatus == chaostypes.DisruptionInjectionStatusNotInjected || !h.disruption.Status.HasTarget(name) {
+				eventType = corev1.EventTypeWarning
```

Comment:
Totaly in tune with you! 👍 
So to be sure to well understand, the idea is to let the ReadinessPrope in warning type by default and just set it to normal type when it's needed?

Reply:
This would be a valid implementation indeed.

The problem I see here, is that when you move the const from one comment zone (warning) to the other (normal) it won't produce any change unless you also change the code down here.

So it's error prone to me.

I noticed we already have a list of Events that is defining the associated eventType for each of them:
* https://github.com/DataDog/chaos-controller/blob/ce27d867bfbf3db0d792f6cc4891cf217625ab00/api/v1beta1/events.go#L71

Can we use this as the default eventType for events in the cache_handler or are this two things not correlated at all 🙈 :
* https://github.com/DataDog/chaos-controller/blob/ce27d867bfbf3db0d792f6cc4891cf217625ab00/controllers/cache_handler.go#L121

I may be wrong here, I was just wondering from the implementation how could we ensure the eventType are properly correlated everywhere if necessary (because we have done it somewhere...)

NB: you may still need to adapt the code in this loop to handle the exception, being from normal to warning. The problem as of today and my understanding is more the default eventType value associated to an eventReason that does not seems at all correlated to the level we define in `Events` and I do not know if it's expected or not TBH

cc @ptnapoleon do you know if those should be correlated?

Reply:
👋 We should definitely use the default event type from the events file in there.
I'm thinking, we could create two different events: EventReadinessProbeChangeDuringDisruption / EventReadinessProbeChangeBeforeDisruption, which would make the event file as the source of truth. 
What do you all think?

Reply:
That would be great to differentiate them more explicitly upfront indeed 🤗 
Could it lead to some breaking changes somewhere?

Reply:
Indeed, it's a good idea! 👍  If we have this kind of events we can remove the function **HasTarget** from **DisruptionStatus**?

Reply:
I don't think there would be any breaking changes.

However, the logic in the PR would have to be moved there: https://github.com/DataDog/chaos-controller/blob/ce27d867bfbf3db0d792f6cc4891cf217625ab00/controllers/cache_handler.go#L234